.devcontainer/db/Dockerfile

```diff
@@ -1,3 +1,3 @@
-FROM postgres:13
+FROM postgres:15.2
```

Comment:
Would it make sense to only use the major version?

Reply:
I put the minor version to be exactly on the same version as AWS, but using the major would be fine too as minor version shouldn't affect the environment consistency that much